oximeter/db/src/client/mod.rs

```diff
@@ -89,7 +93,32 @@ impl Client {
         let client = reqwest::Client::new();
         let url = format!("http://{}", address);
         let schema = Mutex::new(BTreeMap::new());
-        Self { _id: id, log, url, client, schema }
+        Self {
```

Comment:
Maybe have this constructor just delegate to `Self::new_with_request_timeout(address, log, DEFAULT_REQUEST_TIMEOUT)`?

Reply:
Good call!

Reply:
Done in 82f3072109ff6433dad05a371bde8137331b7bff